openedx/adg/lms/static/components/student_account/PasswordResetConfirmation.jsx

```diff
@@ -52,15 +57,21 @@ class PasswordResetConfirmation extends React.Component {
       return (passwordType === 'password') ? 'text' : 'password'
   }
 
+  getEyeIconClasses(prevClasses) {
+      return (prevClasses.includes('show-password')) ? this.hidePasswordClasses : this.showPasswordClasses
```

Comment:
How about defining a variable like `isPasswordVisible = prevClasses.includes('show-password')` and use it in condition to make condition more readable